bndtools.core/test/org/bndtools/core/editors/ImportPackageQuickFixProcessorAddBundleCompletionProposalTest.java

```diff
@@ -1,57 +0,0 @@
-package org.bndtools.core.editors;
```

Comment:
I like tests but I agree I've got an aversion against Eclipse GUI testing. I find I spend way more time testing my own mocks & scaffolding than actual error cases. 

However, if you think, as we discuss later, that you can set up error cases for all the use cases (error codes/source code) then that would be fantastic.

Reply:
I agree - Eclipse testing is tricky. I think writing the tests for the original quick fix processor was a bit of a turning point for me when I started to realise the limitations of mocks - yes, you spend as much time debugging the mocks as you do the actual code. Since then I've had a strong preference for testing against actual code rather than against mocks (where it makes sense to do so), effectively skipping unit testing and going straight for integration tests.

I think the good news is that I've learned a lot more about OSGi and Bnd since those days, and I think I can come up with an OSGi-type test that uses more of actual Eclipse and less mocking. This could become a model for other GUI tests.